app/views/super_admin/content.py

```diff
@@ -71,15 +72,20 @@ def placeholder_upload():
             wpercent = (basewidth / float(img.size[0]))
             hsize = int((float(img.size[1]) * float(wpercent)))
             img = img.resize((basewidth, hsize), PIL.Image.ANTIALIAS)
-            os.mkdir(app.config['TEMP_UPLOADS_FOLDER'])
+            try:
+                os.mkdir(app.config['TEMP_UPLOADS_FOLDER'])
```

Comment:
This puts in try catch but the error still occurs, right? This time it will just pass to except block. So the upload error isn't fixed I guess? Am I missing something?